### DISCUSSION TITLE
Path to Shairport on Rpi

### DISCUSSION THREAD

**Discussion by Taxicletter** [snapcast/snapcast#837] [Q&A]
Hi,
I'm trying to configure a SnapCast Server on a Raspberry Pi Model 3. I think a have the necessary software installed (SnapCast server and Shairport on Raspberry Pi OS lite).
I can see the server in the network, hooray.
But what path do I write in this line:
`airplay:///<path/to/shairport-sync>?name=<name>[&dryout_ms=2000][&devicename=Snapcast][&port=5000][&password=<my password>]`
I'm a beginner to this kind of stuff. I know some basics, but that's it. :-)

**Comment by kingosticks** [comment]
The notation is explained at https://github.com/badaix/snapcast/blob/master/doc/configuration.md and that page also explains the airplay specific parameters. If you search the issues you can see some real people's examples but here are a couple of simple ones:
* airplay:///shairport-sync?name=MyAirPlay
* airplay:///shairport-sync?name=AnotherExample&password=supersecret&port=12345

**Comment by Taxicletter** [comment]
Thank you kingosticks,

I read that page, but I can't find an answer there. Some of the explanations are to complex for my knowledge of linux.

In your examples, its 3 "/", but in the manual it's 2. That doesn't matter? What is right?

So in
`airplay:///<path/to/shairport-sync>?name=<name>[&dryout_ms=2000][&devicename=Snapcast][&port=5000][&password=<my password>]`
I _ignore_ the words "**path/to**" and just write something like in your example. I tried that. If I scan for Airplay devices, I see the name I gave my Raspberry, but tot the name I write in that config file. I did
`sudo systemctl restart snapserver`

Isn't there a step-by-step manual for less advanced users? If there is, you can just point me there, I'm not to lazy to search a solution for myself (if I have enough information).

This is in /etc/snapserver.conf which isn't commented out. That's the default content, except for the airplay stream.
```
[server]
[http]
doc_root = /usr/share/snapserver/snapweb
[tcp]
[stream]
airplay:///shairport-sync?name=LubbaDubMuzik
[logging]
```

**Reply by Taxicletter** [reply]
Maybe this info helps:

In _/etc/init.d_ I can see:  **shairport-sync** and **snapserver** in the list.

I did` systemctl enable shairport-sync` and `systemctl enable snapserver`.

service --status-all:

```
[ + ]  shairport-sync
[ + ]  snapserver
```
In /etc/default/snapserver is nothing then the default:

```
# Start the server, used only by the init.d script
START_SNAPSERVER=true

# Additional command line options that will be passed to snapserver
# note that user/group should be configured in the init.d script or the systemd unit file
# For a list of available options, invoke "snapserver --help" 
SNAPSERVER_OPTS=""
```
